airbyte-ci/connectors/pipelines/pipelines/builds/java_connectors.py

```diff
@@ -11,36 +11,42 @@
 
 
 class BuildConnectorDistributionTar(GradleTask):
+    """
+    A step to build a Java connector image using the distTar Gradle task.
+    """
+
     title = "Build connector tar"
     gradle_task_name = "distTar"
 
+
     async def _run(self) -> StepResult:
-        cdk_includes = ["./airbyte-cdk/java/airbyte-cdk/**"]
-        with_built_tar = (
-            environments.with_gradle(
-                self.context,
-                self.build_include + cdk_includes,
-            )
-            .with_exec(["./gradlew", ":airbyte-cdk:java:airbyte-cdk:publishSnapshotIfNeeded"])
-            .with_mounted_directory(str(self.context.connector.code_directory), await self.context.get_connector_dir())
-            .with_exec(self._get_gradle_command())
-            .with_workdir(f"{self.context.connector.code_directory}/build/distributions")
-        )
+        result = await super()._run()
```

Comment:
Readability: Can we add a comment to remind what's the parent class _run is doing ?

Reply:
Done.